types/licences.d.ts

```diff
@@ -92,5 +92,34 @@ export interface WarningClient {
   raiseWarning: (bookingId: number, code: string, messsage: string) => Promise<void>
   acknowledgeWarnings: (errorIds: number[]) => Promise<number>
   getOutstandingWarnings: () => Promise<List<Warning>>
-  getAcknowledgedWarnings: () => Promise<List<Warning>>
+  getAcknowledgedWarnings: () => Promise<List<Warning>> 
+}
+
+interface ProbationAreas {
+  code: string;
+  description: string;
+}
+
+interface Ldus {
+  code: string;
+  description: string;
+} 
+
+interface activeLdus {
+  code: string;
+  active: boolean
+}
+
+export interface LduService {
+  getAllProbationAreas: () => Promise<Array<Object>>
+  getLdusForProbationArea: (code: string) => Promise<Array<Object>>
+  getActiveLdusForProbationArea: (code: string) => Promise<Array<string>>
+  updateActiveLdus: (probationAreaCode: string, activeLdus) => Promise<string>
```

Comment:
types start with a capital : `ActiveLdus`, then to use this type: `activeLdus: ActiveLdus `

Reply:
done